developer-docs-site/docs/sdks/ts-sdk/key-management.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Key Management"
+slug: "typescript-sdk-key-management"
```

Comment:
please don't introduce slugs, let's just use the file system to designate routes.

Reply:
removed